packages/terra-spacer/tests/jest/SpacerShorthandUtils.test.jsx

```diff
@@ -0,0 +1,85 @@
+import { shorthandValidator, mapShorthandToObject, ShorthandPropErrors } from '../../src/_spacerShorthandUtils';
```

Comment:
nit: unless the jest test matcher is for `*.test.jsx`, this should be `SpacerShorthandUtils.test.js`

Reply:
Done 3aea1be8

Reply:
Done Done 3aea1be